koku/reporting/provider/ocp/models.py

```diff
@@ -1065,16 +1081,16 @@ class Meta:
 
     usage_end = models.DateField(null=False)
 
```

Comment:
All of the changes below are simply a re-organization of the fields. No changes were actually made to the fields.